api/v1alpha08/kogitoserverlessworkflow_types.go

```diff
@@ -391,25 +390,36 @@ type Endpoint struct {
 
 // KogitoServerlessWorkflowStatus defines the observed state of KogitoServerlessWorkflow
 type KogitoServerlessWorkflowStatus struct {
-	Conditions  BuildStatusCondition         `json:"conditions,omitempty"`
-	Endpoints   []Endpoint                   `json:"endpoints"`
-	Address     duckv1.Addressable           `json:"address,omitempty"`
-	Deployments olm.DeploymentStatus         `json:"deployments"`
-	Phase       ConditionType                `json:"phase,omitempty"`
-	Applied     KogitoServerlessWorkflowSpec `json:"applied,omitempty"`
-	Version     string                       `json:"version,omitempty"`
+	Endpoints []Endpoint                   `json:"endpoints,omitempty"`
+	Address   duckv1.Addressable           `json:"address,omitempty"`
+	Condition ConditionType                `json:"phase,omitempty"`
+	Applied   KogitoServerlessWorkflowSpec `json:"applied,omitempty"`
 }
 
 // ConditionType - type of condition
 type ConditionType string
 
 const (
-	// DeployedConditionType - the kieapp is deployed
+	// NoneConditionType --
+	NoneConditionType ConditionType = ""
+	// DeployedConditionType - the workflow is deployed
 	DeployedConditionType ConditionType = "Deployed"
-	// ProvisioningConditionType - the kieapp is being provisioned
+	// ProvisioningConditionType - the workflow is being provisioned
 	ProvisioningConditionType ConditionType = "Provisioning"
-	// FailedConditionType - the kieapp is in a failed state
+	// FailedConditionType - the workflow is in a failed state
 	FailedConditionType ConditionType = "Failed"
+	// WaitingForPlatformConditionType --
+	WaitingForPlatformConditionType ConditionType = "Waiting For Platform"
```

Comment:
What wait on platform means?

Reply:
This state is used when you haven't yet created a platform but you created a Workflow.

The workflow is saved but since you haven't yet created a Platform we miss some information (i.e. registry, secret etc) to start a build and after the build deploy the service.

Reply:
can you please add this in the field description?

Reply:
Updated